src/pjpersist/interfaces.py

```diff
@@ -114,6 +118,20 @@ def get_ghost(coll_name, oid):
         """
 
 
+class IPJConnectionPool(zope.interface.Interface):
+    def getconn():
+        """Get connection from the pool"""
+
+    def putconn(conn):
+        """Put conncetion back to the pool"""
```

Comment:
`putconn` does not make sense to be. Return? Release? "Put" is just the wrong word.

Reply:
I did not invent it. It is part of [psycopg2.pool](https://www.psycopg.org/docs/pool.html#module-psycopg2.pool)